app/data/models/example.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (sequelize, DataTypes) => {
+  return sequelize.define('example', {
+    exampleId: { type: DataTypes.INTEGER, primaryKey: true },
+    textField: DataTypes.STRING
+  },
+  {
+    tableName: 'examples',
+    freezeTableName: true,
+    timestamps: false
+  })
+}
```

Comment:
Just added this as an example to show to how to represent tables as Sequelize models.  Feel free to delete before merging.